amr-wind/convection/incflo_godunov_advection.cpp

```diff
@@ -74,7 +74,7 @@ void godunov::compute_advection(
     Array4<Real> xyzlo = makeArray4(p, bxg1, ncomp);
     p += xyzlo.size();
     Array4<Real> xyzhi = makeArray4(p, bxg1, ncomp);
-    p += xyzhi.size();
+    p += xyzhi.size(); // NOLINT: Value not read warning
```

Comment:
I was thinking we should remove this line.

Reply:
I wasn't certain if `p` was required elsewhere, but I can remove it.